test-shell.sh

```diff
@@ -0,0 +1 @@
+date
```

Comment:
It is recommended to use a temporary directory to generate scripts, or to place files in the testdata directory. 😁